### ISSUE TITLE
Bobtimus should use correct address for cnd

### ISSUE THREAD

**Issue by bonomat** [coblox/bobtimus#79]
At the moment, bobtimus can't retrieve the public ip address of `cnd` if `cnd` is behind a NAT because `cnd` is not aware of that. 

Cnd returned listen addresses on `http://localhost:<port>/` are only locally available addresses, e.g.: 

```
{
  "id": "QmYaEdADq9nZqNWLUXtMP9tnfjLKrwjMBuVSSNyPz2wLLd",
  "listen_addresses": [
    "/ip4/127.0.0.1/tcp/8011",
    "/ip6/::1/tcp/8011",
    "/ip4/192.168.1.36/tcp/8011"
  ]
}
```

Related to: comit-network/comit-rs#1109

**Comment by D4nte** [comment]
shall we add a dependency on https://github.com/comit-network/comit-rs/issues/1109 ?

**Comment by thomaseizinger** [comment]
Closed because #76 adds a configuration option to tackle this problem.
